package.json

```diff
@@ -7,7 +7,8 @@
     "url": "git://github.com/noffle/hypergit.git"
   },
   "bin": {
-    "hypergit": "bin/cli.js"
+    "hypergit": "bin/cli.js",
+    "git-remote-hypergit": "node_modules/.bin/git-remote-hypergit"
   },
```

Comment:
Woah! You can do this!? What a clever hack @ralphtheninja :tada:

Reply:
This might not work on Windows (but that probably isn't a concern yet)